### ISSUE TITLE
Cannot set KOLIBRI_RUN_MODE in options.ini

### ISSUE THREAD

**Issue by nucleogenesis** [learningequality/kolibri#5629]
### Observed behavior

When running Kolibri as a system service in Debian/Ubuntu - I cannot set `KOLIBRI_RUN_MODE` or `RUN_MODE` and have it picked up to override the default value when the server is started.

If I set the `HTTP_PORT=9090` - the server used the new value for `HTTP_PORT` but never took any kind of value for `KOLIBRI_RUN_MODE=msg-[string]`.

Also note - this doesn't work when using either the `RUN_MODE` or `KOLIBRI_RUN_MODE` keys in the options.ini file.

### Expected behavior

When Kolibri starts, it should read all values from `{kolibri-home}/options.ini` and use them in place of any defaults.

### User-facing consequences

Cannot override pingback messages.

### Errors and logs

…

### Steps to reproduce

- Install a *.deb file on Debian.
- Create or edit the `{kolibri_home}/options.ini` file.
- Add the line `KOLIBRI_RUN_MODE=msg-text`
- Start or restart the system service (ie, `sudo systemctl stop kolibri && sudo systemctl start kolibri`)
- In the browser, log in, and you will not see the pingback message.
- Furthermore, you would not see the notice in the Kolibri logs that you are overriding the value for `RUN_MODE`.

### Context

Kolibri Version: v0.12.4
Operating System: Ubuntu 18.04
Browser: Firefox

**Comment by rtibbles** [comment]
Do you have an ENV VAR set (you should) for KOLIBRI_RUN_MODE, when you are doing this? If so, the ENV VAR will always override the options.ini, and this is intended behaviour.

**Comment by nucleogenesis** [comment]
@rtibbles I did not have one set. I was trying to find out why ENV VARs in `~/.bashrc` weren't being used by the daemonized system service and figured I should try to put it into the options.ini file, which didn't work.

I could only get the daemonized service to pickup, specifically, the `KOLIBRI_RUN_MODE` override when I put it into a conf file in `/etc/kolibri`. Also when it was only set in options.ini, the kolibri.log did not show the notice about the `RUN_MODE` being overridden.

**Comment by rtibbles** [comment]
The override notice is only shown when an option is being overridden by an env var, so that's why it wouldn't show up there - but the fact that it's failing to be picked up at all from the options.ini is definitely a worry!

**Comment by jamalex** [comment]
Hmm, so for me it _does_ get picked up from options.ini:

```
$ cat ~/.kolibri/options.ini 

[Deployment]
RUN_MODE=hithere


$ KOLIBRI_RUN_MODE= kolibri shell

In [1]: from kolibri.utils.conf import *

In [2]: OPTIONS["Deployment"]
Out[2]: {'RUN_MODE': 'hithere', 'HTTP_PORT': 8080, 'URL_PATH_PREFIX': '/'}


$ rm ~/.kolibri/options.ini 


$ KOLIBRI_RUN_MODE= kolibri shell

In [1]: from kolibri.utils.conf import *

In [2]: OPTIONS["Deployment"]
Out[2]: {'HTTP_PORT': 8080, 'URL_PATH_PREFIX': '/', 'RUN_MODE': ''}
```

Have you been able to directly verify that it's not picked up, or are you just inferring based on the message not appearing? Another possible explanation for that could be that the message with that ID was already "dismissed", and hence doesn't show anymore. Try changing the value to "msg-somethingelse".

**Comment by rtibbles** [comment]
Is it possible that the `-` is being parsed weirdly inside the options.ini? That's the only difference I am seeing here (aside from the possible message dismissal).

**Comment by nucleogenesis** [comment]
@jamalex @rtibbles 

I'm not sure if this is an issue specifically with the Debian package, but this is on a fresh Ubuntu VM, installed using the 0.12.3-0ubuntu1_all.deb package.

My options.ini:

```
[Deployment]
HTTP_PORT=8090
RUN_MODE=msgjacobini
```

I removed the dash in the `msg-` bit in case Richard's thoughts on the dash were correct.

When I start the server, either using `sudo systemctl start kolibri` or just running `kolibri start` - the HTTP_PORT is always used as it should be - but the RUN_MODE is not.

After the server is started and I log in, I do not see anything in the analytics_pingbacknotifications table in the DB, I see nothing in the log mentioning that I'm overriding RUN_MODE, and I don't get the modal pop-up. I also checked the analytics_pingbacknotificationdismissed table and there are no records there.

**Comment by rtibbles** [comment]
I assume this should only be forwarding any existing env vars, not setting them to blank? https://github.com/learningequality/kolibri-installer-debian/blob/23e2ae064c17c1af7f7729d7dcbbcc5f7f6e6d30/debian/startup/kolibri.default#L75

**Comment by nucleogenesis** [comment]
After some more testing on fresh Ubuntu VMs, it seems that this is no longer an issue in the latest deb release.

When I use the 0.12.3 deb installer from I do not get the message that I expect to get. When I load the kolibri shell - the RUN_MODE is indeed in the options, but for some reason it wouldn't give the pop-up.

However - when I use the 0.12.5 deb installer, everything works as expected without issue.

FWIW - I also tested upgrading from 0.12.3 > 0.12.5 - and it didn't work before the upgrade, but after I upgraded, the message popped up with the RUN_MODE that I set in my {kolibri home}/options.ini.
